### PR TITLE
Feature/import component source

### PR DESCRIPTION
**Pull request by santiagonoguez**
- Loads the component source after fetching
- Includes the DOM4 polyfill that PhantomJS needs to append to DOM
- Create unit test and complete integration test where code is loaded from remote script.

Also, I manually tested it.

### CODE REVIEW

File: test/unit/rise-components-loader.test.js [hunk 1]
```diff
@@ -254,6 +254,38 @@ describe( "ComponentLoader", function() {
...
+        return Promise.resolve({ text: function() {
+          return "RisePlayerConfiguration.testVariable = 2";
+        } });
+      }
+
+      window.addEventListener( "rise-components-loaded", componentsLoadedHandler );
+
+      RisePlayerConfiguration.configure({ playerType: "beta" }, {});
+
+      RisePlayerConfiguration.ComponentLoader.load();
```
Comment (stulees) [inline test/unit/rise-components-loader.test.js]: @santiagonoguez Do you need to call `load()` here? The function will return if in test environment.

File: test/unit/rise-components-loader.test.js [hunk 2]
```diff
@@ -254,6 +254,38 @@ describe( "ComponentLoader", function() {
...
+      function componentsLoadedHandler( event ) {
+        window.removeEventListener( "rise-components-loaded", componentsLoadedHandler );
+
+        expect( event.detail.isLoaded ).to.be.true;
+        expect( RisePlayerConfiguration.testVariable ).to.equal( 2 );
+
+        done();
+      }
+
+      function successfulDownload() {
```
Comment (stulees) [inline test/unit/rise-components-loader.test.js]: @santiagonoguez Just to clarify, the main difference in this unit test to the integration test is the unit test mocks the `window.fetch` here, correct?

Comment (santiagonoguez) [reply test/unit/rise-components-loader.test.js]: @stulees yes, the integration test does a real fetch to GCS

File: test/unit/rise-components-loader.test.js (hunk 1 above)
Comment (santiagonoguez) [reply test/unit/rise-components-loader.test.js]: @stulees it's not needed indeed here, thanks ! I've removed it to avoid confusion.

**Pull request merged**